database_setup/aggregated-arrivals-column-add.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE public.arrival ADD COLUMN scheduled_departure timestamp;
```

Comment:
I'm thinking since we don't have a mechanism for migrations we should also add this column to the `CREATE TABLE` statement in `aggregated_arrivals.sql` so we can create the right structure in one step for future local or server environments.

Reply:
👍